packages/fxa-settings/src/components/LinkRememberPassword/index.stories.tsx

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import LinkRememberPassword from '.';
+import { LocationProvider } from '@reach/router';
+import { Meta } from '@storybook/react';
+import { MOCK_EMAIL } from './mocks';
+
+export default {
+  title: 'components/LinkRememberPassword',
+  component: LinkRememberPassword,
+} as Meta;
+
+export const Default = () => (
```

Comment:
You could add that `forceEmail` state here too named `WithForceEmail` and rename `Default` to `WithEmail` or something like it. Also good to show the state (and/or test) if neither prop is provided. If one of these is always required but you can't have both, or whatever the case is, the type could be a little nicer possibly, see [this SO answer](https://stackoverflow.com/questions/37688318/typescript-interface-possible-to-make-one-or-the-other-properties-required) or another similar one.

Reply:
In this case, it's none or one (but not both).

Reply:
@vpomerleau I think you can use unions as shown in that SO link but I won't block r+ for it.